ament_python/data_files.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Override the setup.py command line argument --script_dir."""
+
+import inspect
+import os
+
+
+def get_data_files(package_name, *, marker_file=None):
+    """Return a dictionary containing the data_files of the package."""
+    data_files = []
+
+    # install package manifest
+    data_files.append(('share/' + package_name, ['package.xml']))
```

Comment:
Since this library and function are to be used for explicitly ament_python rather than pure python packages there should indeed always be a  package.xml.